example_files/visualization/input_visualization_feature.html

```diff
@@ -444,31 +446,80 @@ <h2>Annual End Uses</h2>
           var localEndUseMax = 0;
 
           if(feature['complete_simulation'] == true){
+
             // monthly fuel use
             $scope.monthlyFuelChartData[feature.name] = [];
+            var changeToKbtu = false;
+            var kbtu_datasets = ['Gas:Facility', 'Propane:Facility', 'FuelOil#2:Facility', 'OtherFuels:Facility'];
+            // first iterate through all datasets to see if you'll need to change to kBtu units
             _.forEach(datasets, function (dataset) {
+              if (kbtu_datasets.indexOf(dataset) != -1) {
+                changeToKbtu = true;
```

Comment:
@kflemin I wanted to confirm - the `datasets` variable does not change dynamically based on the scenarioData.js file. Also the scenarioData.js would be reporting the residential fuel types even if there is no consumption but would put 0 for those monthly values.
Would this if statement return `true` always in this case? since these kbtu_datasets are present in the `dataset` array?

Reply:
ah, yeah we need to change this then. you should check that it is in datasets and not all 0s then. good catch!

Reply:
FWIW Joe pointed out at bug that was introduced to reporting-gem when we switched to using OpenStudio 3.1: `Gas` changed to `NaturalGas` so until his PR gets merged into reporting-gem we should expect `Gas` to always return zeroes.

Reply:
can we add tests for this so we can catch when a key changes name?  thanks!

Reply:
also @vtnate, is this just for residential workflow that gas will return all 0s or for all workflows?

Reply:
I presume it's for all workflows.

It's not obvious to me how we'd check for changed output from OS. Be more careful when we change versions, to check output from runs before and after?

Reply:
[OS 9.3](https://bigladdersoftware.com/epx/docs/9-3/input-output-reference/input-for-output.html#outputmeter-and-outputmetermeterfileonly) schema vs [OS 9.4](https://bigladdersoftware.com/epx/docs/9-4/input-output-reference/input-for-output.html#outputmeter-and-outputmetermeterfileonly) schema. Scroll down to the first table to see `Gas` replaced with `NaturalGas`.

Reply:
Maybe I'll make a separate reporting-gem PR just to fix this bug. It should be a one-liner.

Reply:
@vtnate If you're adding that in, I can change to `NaturalGas` here too

Reply:
@tanushree04 I'll just change the mapping in reporting-gem, you shouldn't have to do anything here.

Reply:
There is an array of fuel types, I would need to change it there. It looks for those key words in the feature reports.

Reply:
I changed the mapping with [this reporting-gem PR](https://github.com/urbanopt/urbanopt-reporting-gem/pull/65). I'm not sure this is a complete fix yet though. Hopefully the feature reports won't change keys.

Reply:
@vtnate I tested with this branch, the feature keys did not change. I did not see any Gas energy consumption in the feature report though, even though I see it in the eplus report. Do we have to change this at more instances?